resources/prosody-plugins/util.lib.lua

```diff
@@ -260,11 +280,17 @@ end
 
 -- healthcheck rooms in jicofo starts with a string '__jicofo-health-check'
 function is_healthcheck_room(room_jid)
+    local ret = hc_room_cache:get(room_jid);
+    if ret then
+        return ret;
+    end
     if starts_with(room_jid, "__jicofo-health-check") then
```

Comment:
Is it really worth optimizing a `starts_with`?

Reply:
maybe not :) I think I saw it in some profiling ... and that's why I added it ... 
Maybe I will drop it and will see how it goes at a later stage.